test/index.test.js

```diff
@@ -37,4 +37,17 @@ describe('zstd', () => {
       });
     });
   });
+
+  describe('#decompress', () => {
+    context('when decompressing invalid data', () => {
+      it('includes a stack trace', async () => {
+        try {
+          await decompress(Buffer.from('invalid'));
+        } catch (error) {
+          expect(error.message).to.be('zstd: Unknown frame descriptor');
+          expect(error.stack).to.match(/at decompress/);
```

Comment:
Stack will output with these changes:
```
Error: zstd: Unknown frame descriptor
    at module.exports.decompress (/Users/modetojoy/work/mongodb-js/zstd/index.js:167:11)
    at async Context.<anonymous> (/Users/modetojoy/work/mongodb-js/zstd/test/index.test.js:45:11)
```

```suggestion
          expect(error.stack).to.match(/module.exports.decompress/);
```